docs/src/main/paradox/buildtools/gradle.md

```diff
@@ -13,7 +13,7 @@ By default both client and server are generated and Java or Scala is autodetecte
 
 ### Installation
 
-Follow instructions at [Gradle plugin portal](https://plugins.gradle.org/plugin/com.lightbend.akka.grpc.gradle) to apply plugin.
```

Comment:
Hi! :wave: :smile: 

I don't see a way to edit the instructions at https://plugins.gradle.org/plugin/com.lightbend.akka.grpc.gradle from the web UI, I suspect they will get updated automatically by changing the fields under `pluginBundle` in `gradle-plugin/build.gradle`.

Of course it would still be good to transfer ownership of this package to you. Looks like this can be done by opening an issue at https://github.com/gradle/plugin-portal-requests#transferring-ownership . Can you create an account to transfer ownership to?

Reply:
Heyhey, I've created an account `johanandren`. Please trigger the transfer thanks!

Reply:
Thanks @raboof !